### ISSUE TITLE
Reconsider way balloon position updates are provided to AT

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#304]
With all due respect to @jessegreenberg, the way alerts about how the balloon position changes are sent to screen readers has has become a horrifying mess. It should be updated and improved, and I don't think it should all be in a callback for BalloonModel.positionProperty.

**Comment by terracoda** [comment]
@jessegreenberg, I'm not sure what the exact issue is here. 
However, Ashley had several issues when using JAWS today with both IE 11 and FF (fresh install) with JAWS. I am opening issues for those items.

**Comment by jessegreenberg** [comment]
The issue is that these alerts are sort of working, but they are a house of cards. Thanks for making new issues.

**Comment by jessegreenberg** [comment]
I started exploring some cleanup in the  above  branch but it broke too many things to merge into master for now.  Overall, improvements were made by linking things more directly to axon/Properties instead of ```step```, I would like to continue with that.

**Comment by jessegreenberg** [comment]
I am much happier with this now and changes are in master, the branch can be deleted.

**Comment by jessegreenberg** [comment]
Branch deleted. Closing.

**Comment by terracoda** [comment]
@jessegreenberg, a quick check sounded good. Will listen more tomorrow. You are awesome!
